packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -31,6 +28,8 @@ const MainTitleBar = () => {
 
     return (
         <React.Fragment>
+            <WalletsBanner />
+            <BookBanner />
             {show_wallets_banner && <WalletsBanner />}
```

Comment:
@aswathy-deriv Two WalletsBanner?

Reply:
Hi @sergei-deriv , can u please help clarify the required code. This was done wrt https://github.com/binary-com/deriv-app/pull/14299/files#diff-ae6e89116c7396dc88f2d7e3dd3f1a65d380dd9c1dfcd063f1f477493a21828d

Reply:
Should both the banners be lazy loaded?

Reply:
@likhith-deriv When Nikita created this PR lazy load part was not merged yet, but now in the master we have lazy load and showing `WalletsBanner` by checking `show_wallets_banner`